content/docs/practices/instrumentation.md

```diff
@@ -32,6 +32,9 @@ this category.
 The key metrics in such a system are the number of performed queries, errors,
 and latency. The number of in-progress requests can also be useful.
 
+When exposing operation counters, create seperate metrics for total and failed number of
+requests. That makes calculating percental error rates easier.
```

Comment:
I suggest to add the following at the end of the section (after the paragraph about counting queries):

For counting failed queries, see section [Failures](#failures) below.